### ISSUE TITLE
Merging of properties with different values that are separated.  

### ISSUE THREAD

**Issue by tomkowalski** [clean-css/clean-css#810]
Running cssmin on: 

```
#main {
    display: flex;
    clear:both;
    display: -webkit-flex;
}
```

will remove display: flex;
(note: turning off agressive-reconstuction will stop this.)

While this can be remedied with the flag, i dont think this is intended behaviour and can cause unwanted bugs if properties arent adjacent.

**Comment by jakubpawlowicz** [comment]
We'd like to have all options opt-in rather than opt-out in the future, so 👍.

**Comment by jakubpawlowicz** [comment]
Depends on #623.

**Comment by jakubpawlowicz** [comment]
Won't fix as clean-css is going into [maintenance mode](https://github.com/clean-css/clean-css/discussions/1209).
